src/core/test/system/users/whitelisted_investor.sol

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2020 Centrifuge
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity >=0.5.12;
+
+import "ds-test/test.sol";
+import "../../../lender/deployer.sol";
+import "../test_utils.sol";
```

Comment:
maybe you can also add the interfaces you need to the interfaces file in the test folder and import it, then you can remove the test_utils & deployer imports.

Reply:
good idea, I will add it in the TestUtils Pr